layers/meta-resin-raspberrypi/recipes-support/hostapp-update-hooks/hostapp-update-hooks.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+
+HOSTAPP_HOOKS += " 99-rpi-bootloader"
```

Comment:
NIT: extra space here

Reply:
Correct. I'll leave that though as we need to release asap. Thanks for pointing it out.